src/docfx/lib/log/Telemetry.cs

```diff
@@ -151,12 +151,24 @@ public static void TrackExternalLink(string tag, string attribute, string scheme
             }
         }
 
-        public static void TrackErrorCount(string code, ErrorLevel level, string? name)
+        public static void TrackErrorCount(Error error)
         {
+            var code = error.Code;
+            var level = error.Level;
+            var name = error.PropertyPath;
+            var additionalInfo = new
+            {
+                error.MsAuthor,
+                error.MsProd,
+                error.MsTechnology,
+                error.MsService,
+                error.MsSubservice,
+            };
+            var additoinInfoString = JsonUtility.Serialize(additionalInfo);
             if (!s_isRealTimeBuild.Value)
             {
                 s_errorCountMetric.TrackValue(
-                    1, code, level.ToString(), CoalesceEmpty(name), "User", s_os, s_version, s_repo, s_branch, s_correlationId, s_sessionId);
+                    1, code, level.ToString(), CoalesceEmpty(name), additoinInfoString, s_os, s_version, s_repo, s_branch, s_correlationId, s_sessionId);
```

Comment:
Is it okay to remove type = "User" here?

Reply:
Or we can include 'User' into above additionalInfo object? If yes, I think we need to keep its name as it is.

Reply:
Offline talked to @fkpwolf , he has confirmed with @yufeih that 'User' is hard-coded, and not used. So we can discard it.